packages/jest-environment-ceramic/index.js

```diff
@@ -17,16 +17,19 @@ export default class CeramicEnvironment extends NodeEnvironment {
         },
         Pubsub: {
           // default "gossipsub" uses CJS and fails to import
-          Router: 'floodsub',
+          PubSubRouter: 'floodsub',
         },
       },
-      profiles: ['test'],
       repo: path.join(this.tmpFolder.path, 'ipfs'),
       silent: true,
     })
+    const stateStoreDirectory = path.join(this.tmpFolder.path, 'ceramic')
     this.global.ceramic = await Ceramic.create(this.global.ipfs, {
-      anchorOnRequest: false,
-      stateStoreDirectory: path.join(this.tmpFolder.path, 'ceramic'),
+      stateStoreDirectory: stateStoreDirectory,
+      indexing: {
+        db: `sqlite://${stateStoreDirectory}/ceramic.sqlite`,
+        models: []
```

Comment:
That's good defaults but could you make these configurable please? see https://github.com/ceramicstudio/js-glaze/blob/main/packages/jest-environment-glaze/index.js#L8-L12